docs/getting_started.md

```diff
@@ -68,14 +68,15 @@ Fog::OpenStack.services
 These services can be explored in further depth in the following documents:
 
 * [Compute](compute.md)
+* [Introspection](introspection.md)
+* [Metering](metering.md)
+* [Network](network.md)
+* [NFV](nfv.md)
+* [Orchestration](orchestration.md)
+* [Planning](planning.md)
+* [Share (Manila)](share.md)
 * [Storage (Swift)](storage.md)
+* [Workflow](workflow.md)
 
 ## Additional Resources
-* [fog.io](http://fog.io)
-* [Fog rdoc](http://rubydoc.info/gems/fog)
-* [Fog Github repo](https://github.com/fog/fog)
-* [Release Notes](https://github.com/fog/fog/blob/master/changelog.txt)
-
-
-## Support and Feedback
-Your feedback is appreciated! If you have specific issues with the **fog** SDK, you should file an [issue via Github](https://github.com/fog/fog/issues).
+[resources and feedback](shared/resources.md)
```

Comment:
Same as above, please submit a separate PR for those changes not related to your initial Manila commit.
Also, I think, `shared` is confusing I would opt for `common` but it's only my view of it. :)